packages/commonwealth/client/scripts/views/pages/communities.tsx

```diff
@@ -135,7 +136,14 @@ const CommunitiesPage = () => {
     return res;
   };
 
-  const sortedCommunities = sortCommunities(app.config.chains.getAll());
+  const activeCommunities = useFetchActiveCommunitiesQuery();
```

Comment:
`  const { data: activeCommunities } = useFetchActiveCommunitiesQuery();`

it's kinda of convention we use. Also, you can add `{ isLoading: isActiveCommunitiesLoading }` if we need spinner